src/GenAPI/Microsoft.DotNet.GenAPI/CSharpSyntaxWriter.cs

```diff
@@ -0,0 +1,113 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.DotNet.GenAPI
+{
+    public class CSharpSyntaxWriter : CSharpSyntaxRewriter
```

Comment:
Please add triple slash comments to the class and add `<inheritdoc />` statements to the public override methods below.

Reply:
Updated:
```
    /// <summary>
    /// Represents a <see cref="CSharpSyntaxVisitor{TResult}"/> which descends an entire <see cref="CSharpSyntaxNode"/> graph and
    /// modify visited constructor, class, interface, struct, method declarations SyntaxNodes in depth-first order.
    /// </summary>
```